rpcserver.go

```diff
@@ -526,17 +526,19 @@ func (s *rpcServer) InitAccount(ctx context.Context,
 		value := btcutil.Amount(req.AccountValue)
 		confTarget := req.GetConfTarget()
 
-		feeRate, _, err := s.accountManager.QuoteAccount(
+		feePerKw, _, err := s.accountManager.QuoteAccount(
```

Comment:
I think the shadowing should be fixed to make the code more easy to read.
So just:
```
var err error
feeRate, _, err = s.accountManager.QuoteAccount(
```